<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>480</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>1000</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>970</width>
    <height>725</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Маятник Фуко</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(92, 105, 141);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="minimumSize">
    <size>
     <width>800</width>
     <height>480</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000000</width>
     <height>1000000</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>970</width>
     <height>725</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_place">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Расположение маятника</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_north">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Северный полюс</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_equator">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Экватор</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radio_south">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Южный полюс</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>246</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_place_2">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Дополнительные настройки</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="isAcceleration">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Ускорить вращение Земли</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="isTrajectory">
          <property name="font">
           <font>
            <family>Times New Roman</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Отображать траекторию</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>244</width>
          <height>13</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="start_btn">
          <property name="minimumSize">
           <size>
            <width>187</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>400</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    color: #333;
    border-radius: 20px;
    border-style: outset;
    padding: 5px;
	background-color: rgb(85, 170, 255);
    }

QPushButton:hover {
    background-color: rgb(85, 190, 255);
    }

QPushButton:pressed {
    border-style: inset;
    background-color: rgb(85, 190, 255);
    }

QPushButton:disabled {
	color: #777;
    border-style: inset;
    background-color: rgb(38, 75, 113);
    }</string>
          </property>
          <property name="text">
           <string>Запуск</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="stop_btn">
          <property name="minimumSize">
           <size>
            <width>187</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>400</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    color: #333;
    border-radius: 20px;
    border-style: outset;
    padding: 5px;
	background-color: rgb(85, 170, 255);
    }

QPushButton:hover {
    background-color: rgb(85, 190, 255);
    }

QPushButton:pressed {
    border-style: inset;
    background-color: rgb(85, 190, 255);
    }

QPushButton:disabled {
	color: #777;
    border-style: inset;
    background-color: rgb(38, 75, 113);
    }</string>
          </property>
          <property name="text">
           <string>Пауза</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="reset_btn">
          <property name="minimumSize">
           <size>
            <width>187</width>
            <height>70</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>400</width>
            <height>400</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    color: #333;
    border-radius: 20px;
    border-style: outset;
    padding: 5px;
	background-color: rgb(85, 170, 255);
    }

QPushButton:hover {
    background-color: rgb(85, 190, 255);
    }

QPushButton:pressed {
    border-style: inset;
    background-color: rgb(85, 190, 255);
    }

QPushButton:disabled {
	color: #777;
    border-style: inset;
    background-color: rgb(38, 75, 113);
    }</string>
          </property>
          <property name="text">
           <string>Остановить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="minimumSize">
       <size>
        <width>950</width>
        <height>934</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>950</width>
        <height>934</height>
       </size>
      </property>
      <property name="baseSize">
       <size>
        <width>710</width>
        <height>710</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">border: 2px solid #555;
border-color: rgb(85, 0, 255);</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
